.github/workflows/ci.yaml

```diff
@@ -44,6 +45,8 @@ jobs:
       with:
         ruby-version: "${{ matrix.ruby-version }}"
         bundler-cache: true
+        # fix for ovirt-engine-sdk (>4.6.0 should make this not necessary)
+        gem-install-options: '--with-cflags="-Wno-error=incompatible-function-pointer-types -Wno-error=implicit-function-declaration"'
```

Comment:
While this works, I think this should live in bin/before_install in a CI only section (or perhaps in a general section, but then I think we need platform specific choices)

Reply:
This is necessary for all? developers
I was not able to get this working with bundler configuration.

Also, not sure how to do this outside of here since bundler will not be present before this step.

Reply:
but thanks - works fine in CI